### ISSUE TITLE
Don't require Chrome to be installed

### ISSUE THREAD

**Issue by ryandesign** [kopurando/virga.app-youtube-dl-GUI-for-OS-X#9]
Launching Virga on a system that doesn't have Google Chrome installed causes the standard AppleScript dialog "Where is 'Google Chrome'?" to appear. It would be great if that didn't happen.

**Comment by noyannus** [comment]
Yes, please find a way to use the standard browser, or be completely browser-independent.
